src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectMessage.cs

```diff
@@ -280,6 +304,20 @@ public string DomainHint
         }
 
         /// <summary>
+        /// Gets or sets whether parameters for the library and version are sent on the query string for this <see cref="OpenIdConnectMessage"/> instance. 
+        /// This value is set to the value of EnableTelemetryParametersByDefault at message creation time.
+        /// </summary>
+        public bool EnableTelemetryParameters
```

Comment:
The only thing I can think of is if we did

{get;set;} = EnableTelemetryParametersByDefault;

would remove the need for a few sets in some methods.